CHANGELOG.md

```diff
@@ -51,7 +51,7 @@ Here is a template for new release sections
 - `OPTIMIZED_ADD_CAP` replaced by "optimized_add_cap" in the assets keys returned to EPA after simulating (#939)
 - The if statement for adapting `MAXIMUM_CAP` for non-dispatchable production assets is now based on the value of `DISPATCHABILITY` and not on the existence of the key `FILENAME` in the `asset_dict` (#939)
 - The default values for the constraints are now located in `src/constants.py` under the variable `DEFAULT_CONSTRAINT_VALUES` (#953)
-
+- When a required parameter is missing its default value, defined in `docs/MVS_parameters_list.csv`, is used instead of raising an error (#956)
```

Comment:
```suggestion
- When a required parameter is missing a value, a default value defined in `docs/MVS_parameters_list.csv` is used instead of raising an error (#956)
```